plugin-dev/Source/Sentry/Public/SentrySettings.h

```diff
@@ -285,6 +285,10 @@ class SENTRY_API USentrySettings : public UObject
 		Meta = (DisplayName = "Sentry database location", ToolTip = "Location where Sentry stores its internal/temporary files."))
 	ESentryDatabaseLocation DatabaseLocation;
 
+	UPROPERTY(Config, EditAnywhere, Category = "General|Native",
```

Comment:
Should we have this `true` by default?
If we can detect it's running on docker ideally, perhaps there are some heuristics?

I mean Linux is mostly for servers (outside SteamOS?) and Windows if it's Windows Server, it's also fine to be `true` by default.